pkg/devfile/generator/utils_test.go

```diff
@@ -127,12 +128,14 @@ func TestConvertPorts(t *testing.T) {
 			},
 			want: []corev1.ContainerPort{
 				{
-					Name:          endpointsNames[0],
+					Name:          "8080-tcp",
```

Comment:
Just noticed, can you update teh test case name, it seems wrong

Reply:
updated